GSA-TTS/FAC
# Correcting PASSTHROUGHAWARD transformation tracking

Follow-up of https://github.com/GSA-TTS/FAC/pull/3762

In this PR:
- Now creating inspection records for all federal awards of an audit as long as at least one of them requires a transformation for the `PASSTHROUGHAWARD` field

Testing:
- Submitting an audit via app should pass cross validation
- "Y" case
  - Find an audit in the `elecaudits` table with a blank `PASSTHROUGHAWARD`, or clear one that already exists, and has a `PASSTHROUGHAMOUNT`
  - Run `historic_data_migrator` on that audit and it should migrate without issue
  - `is_passthrough_award` in the dissemination_federalaward table should be "Y"
  - The entry in `migrationinspectionrecord` should show it replaced "" with "Y", while other `PASSTHROUGHAWARD` values remain as-is
- "N" case
  - Find an audit in the `elecaudits` table with a blank `PASSTHROUGHAWARD`, or clear one that already exists, and has no `PASSTHROUGHAMOUNT`
  - Run `historic_data_migrator` on that audit and it should migrate without issue
  - `is_passthrough_award` in the dissemination_federalaward table should be "N"
  - The entry in `migrationinspectionrecord` should show it replaced "" with "N", while other `PASSTHROUGHAWARD` values remain as-is

## PR checklist: submitters


## PR checklist: reviewers


The larger the PR, the stricter we should be about these points.
